File: webapp/HIN_WEB/ClientApp/src/app/model/bulkassign.ts
export class Bulkassign {
  public LeadFunnelStatusId: number;
  public SecurityGroup: number;
  public AssignedToId: number;
  public AssignedToName: string;
  public ArrayOfData: Array<AssignedInput> = [];
  public WonActualAmount: number;
  public Reason: string;
}

export class AssignedInput {
  public Id: number;
  public EntityType: number;
}

File: webapp/HIN_WEB/Models/Bulkassign.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace HIN_WEB.Models
{
    public class Bulkassign
    {
        public int? LeadFunnelStatusId { get; set; }
        public int? SecurityGroup { get; set; }
        public int? AssignedToId { get; set; }
        public string AssignedToName { get; set; }
        public List<AssignedInput> ArrayOfData { get; set; }
        public int? WonActualAmount { get; set; }
        public string Reason { get; set; }
    }
}
